### ISSUE TITLE
Features overriden??

### ISSUE THREAD

**Issue by duartegarin** [govCMS/GovCMS7#555]
Hi guys,
I am starting a GovCMS for a government client and I have a couple of questions regarding customisation.

We need to make some changes to the default installation and it seems most is highly managed through features. The first issue is that on a fresh install features are overriden (no changes yet). The second (and main) issue is that we need to change some features and don't want to risk loosing our changes in the upgrade process. 

As an example I might want to add/remove fields from a Publication (which overrides the feature). What's the common practice for this? I also notice that there a huge dependency between features which is not a great practice. Everything is required by 'GovCMS' so you can't disable features without changing the info files and removing those dependencies and, again, that would be lost in an update process.

What's the recommended upgrade process given that it seems almost impossible to not have an overridden features list?

Cheers,

**Comment by TracyStyles** [comment]
Hello,

Apologies for the delay in providing a response. We have been short staffed in the office this week. I will arrange for one of our technical support members to look into this next week (after the Labour Day Public Holiday).

Thank you for your patience.

Tracy

**Comment by duartegarin** [comment]
Hi @TracyStyles ,
Thanks for getting back to me on this.
Will wait your further feedback.

Thanks,
Duarte

**Comment by invisigoth** [comment]
Hi duartegarin

The features shipped with govCMS are more of a legacy of the old distribution. It is my understanding that we do no revert or recreate any feature in the govCMS SaaS environment.

Therefore it is safe for you to just disregards the overridden feature messages and do whatever you need to do to the content types etc. 

Please also be advised that you cannot use features to transfer any of your configuration because govCMS SaaS does not allow custom modules to be included.

In short words, for govCMS SaaS customers, please do not use features and just disregards the overridden feature messages.

**Comment by duartegarin** [comment]
Hi @invisigoth ,
This project in is PaaS so we can override it easily.
I just want to make sure that there isn't any risk of loosing changes if we update.
Also as I asked, how do you deal with the cyclic dependencies you have in the distro? The base depends on everything so there is no way to "turn off" certain features without changing the info files of the govcms distro.
Can you advise?
Thanks,
Duarte
